cryptography/src/lib.rs

```diff
@@ -124,6 +124,42 @@ pub trait Hasher: Clone + Send + Sync + 'static {
     fn random<R: Rng + CryptoRng>(rng: &mut R) -> Digest;
 }
 
+pub trait BatchScheme {
```

Comment:
added a `BatchScheme`, it might be subject to changes when implementing BLS12381 batch.